bat-utils/lib/runtime-currency.js

```diff
@@ -302,9 +305,11 @@ const monitor3 = (config, runtime) => {
 
 const maintenance = async (config, runtime) => {
   const now = underscore.now()
-  let fxrates, results, tickers
+  let fxrates, rates, results, tickers
 
   if (config.helper) {
+    rates = JSON.parse(JSON.stringify(singleton.rates))
```

Comment:
?

Reply:
that is the "easy" way to deep-clone a non-cyclic javascript object. i can require underscore-deepclone if you prefer.